notebooks/official/generative_ai/ai21labs_intro.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>"To use an AI21 on Vertex AI" &gt; "To use an AI21 <strong>model</strong> on Vertex AI"</p><p> </p><p>"You can stream your AI21 models responses" &gt; "You can stream your model responses"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>